projects/storefrontlib/shared/components/list-navigation/pagination/pagination.component.ts

```diff
@@ -61,6 +61,21 @@ export class PaginationComponent {
     );
   }
 
+  /**
+   * Format aria-label based on pagination page type.
+   *
+   * @param label
+   * @param item
+   * @returns
+   */
+  getAriaLabel(label: string, type: PaginationItemType) {
```

Comment:
Good practice to always have the return type.

```suggestion
  getAriaLabel(label: string, type: PaginationItemType): string {
```

Reply:
Fixed